README.md

```diff
@@ -88,6 +88,12 @@ pi@address ~ $ bash init-pi.sh
 ```
 <br />
 
+### Set up Artik053 board to test
+
+In case of the artik053 devices, the communication happens over the serial port. You only need a `microusb` cable in order to test and need to change the serial device id using `--port /dev/ttyUSBx` as testrunner parameters. If you want to fix this, please refer to the `STM32F4-Discovery` board guide above.
```

Comment:
Could you please change the last sentence (to be more clear)?

For example:
To handle the device without root permission, please follow the `STM32F4-Discovery` [flash](https://github.com/Samsung/js-remote-test#flash-the-device-without-root-permission) and  [read/write](https://github.com/Samsung/js-remote-test#readwrite-serial-without-root-permissionn) sections.

Probably, in this case the `--port /dev/ttyUSBx` part could be removed.

Reply:
The [examples](https://github.com/Samsung/js-remote-test#examples-to-run-tests) section should be updated.

Reply:
I updated it.